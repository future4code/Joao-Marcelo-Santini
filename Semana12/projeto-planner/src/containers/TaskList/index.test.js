import React from "react";
import renderer from "react-test-renderer";
import { TaskList } from "./index";

describe("Teste TaskList", () =>{
    test("Snapshot TaskList", () => {
        const snapTaskList = renderer.create(<TaskList tasks={[]} getTasks={()=>{}} />)
        .toJSON();
        expect(snapTaskList).toMatchSnapshot();
    });

});