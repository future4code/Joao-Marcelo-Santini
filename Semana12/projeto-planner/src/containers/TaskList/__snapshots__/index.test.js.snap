// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Teste TaskList Snapshot TaskList 1`] = `
<div
  className="sc-bdVaJa cotxIj"
>
  <div
    className="sc-bwzfXH FdJee"
  >
    <h3>
      Segunda-Feira
    </h3>
  </div>
  <div
    className="sc-bwzfXH FdJee"
  >
    <h3>
      Terça-Feira
    </h3>
  </div>
  <div
    className="sc-bwzfXH FdJee"
  >
    <h3>
      Quarta-Feira
    </h3>
  </div>
  <div
    className="sc-bwzfXH FdJee"
  >
    <h3>
      Quinta-Feira
    </h3>
  </div>
  <div
    className="sc-bwzfXH FdJee"
  >
    <h3>
      Sexta-Feira
    </h3>
  </div>
  <div
    className="sc-bwzfXH FdJee"
  >
    <h3>
      Sábado
    </h3>
  </div>
  <div
    className="sc-bwzfXH FdJee"
  >
    <h3>
      Domingo
    </h3>
  </div>
</div>
`;
